CHANGES.md

```diff
@@ -53,3 +53,5 @@ Please follow the [`CHANGES.md` modification guidelines](https://github.com/dcos
 * Marathon apps have support for CSI volumes. (MARATHON-8765)
 
 * Stop all services at once during upgrade. (COPS-6512)
+
+* Mark inactive service as unhealthy. (COPS-6514)
```

Comment:
This is only the Telegraf service that is now marked unhealthy if inactive? The changelog should name the specific service. Do we need to do the same for other services?

Reply:
all inactive services will be marked as unhealthy